server.js

```diff
@@ -73,6 +73,7 @@ app.use(compression())
 app.use(
   helmet({
     contentSecurityPolicy: false,
+    referrerPolicy: false,
```

Comment:
I'm not very familiar with this header. Is there a difference between disabling it with `false` and using `referrerPolicy: { policy: "no-referrer" }`?

Reply:
Seems like [most browsers use `no-referrer-when-downgrade`](https://web.dev/referrer-best-practices/), and Chrome has `strict-origin-when-cross-origin` as of Chrome 85.

It works for me on Chrome and Safari, so I think this is good to go @HashimotoYT!